salesagility/SuiteCRM
# US 1.9

Added JIM's Business Hours Package
## Description
Removed Existing Business hours UI from Configurator module.
Added JIM's Business Hours Package to Administrator Module.
Added BusinessHours Module to core.
Changed task end date calculation in projects and project templates module.

## How To Test This
Navigate to Admin->System settings. Verify that there are no Business Hour settings.
Navigate to Admin, there should be Business Hours setting option. click and verify you see dropdown for each day's working hours.
Navigate to Projects Module, open a project and navigate to gantt chart. edit or ad a task and verify it considers or ignores business hours defined based on "consider business hours" checkbox.

## Types of changes


### Final checklist






As a User I want the 'Override Business Hours' field renamed to
'Consider Working days'.
